ember/app/ui/components/identity-addresses/component.js

```diff
@@ -22,6 +21,7 @@ export default class IdentityAddressesComponent extends Component {
   @service notification;
   @service store;
   @service intl;
+  @service fetch;
```

Comment:
this would change your call from `fetch` to `fetch.fetch`. I think some file changes needed here and in the other files.

Reply:
Done

Reply:
Not yet pushed ? :sweat_smile:

Reply:
Yes, now pushed